src/main/java/net/javadiscord/javabot/systems/commands/SearchCommand.java

```diff
@@ -77,8 +77,13 @@ public ReplyAction handle(SlashCommandEvent event) {
                 name = object.get("name").getAsString();
                 url = object.get("url").getAsString();
                 snippet = object.get("snippet").getAsString();
-                if (object.get("snippet").getAsString().length() > 260) {
-                    snippet = object.get("snippet").getAsString().substring(0, 260).concat("...");
+                if (object.get("snippet").getAsString().length() > 320) {
+                    snippet = object.get("snippet").getAsString().substring(0, 320);
+                    int snippetLastPeriod = snippet.lastIndexOf('.');
+                    if (snippetLastPeriod != -1)
+                        snippet = snippet.substring(0, snippetLastPeriod + 1);
+                    else
+                        snippet = snippet.concat("...");
```

Comment:
For multi-line if-statements, please add curly brackets. Makes the whole thing more readable